vts/policymanager/policymanager.go

```diff
@@ -67,7 +67,7 @@ func (o *PolicyManager) getPolicy(ev *proto.EvidenceContext) (*policy.Policy, er
 
 	vals, err := o.Store.Get(policyID)
 	if err != nil {
-		return nil, err
+		return nil, ErrNoPolicy
```

Comment:
the only problem is we lose information about the root cause.

what about:
```suggestion
		return nil, fmt.Errorf("%w: %v", ErrNoPolicy, err)
```

And if the caller needs to branch based on the error type they still can do that:
```
        if errors.Is(err, ErrNoPolicy) {
                // specific handling for "no policy"
        } else {
                // generic error handling
        }
```

Reply:
That should work